paseos/actors/base_actor.py

```diff
@@ -158,19 +177,29 @@ def get_position_velocity(self, epoch: pk.epoch):
         return self._orbital_parameters.eph(epoch)
 
     def is_in_line_of_sight(
-        self, other_actor: "BaseActor", epoch: pk.epoch, plot=False
+        self,
+        other_actor: "BaseActor",
+        epoch: pk.epoch,
+        minimum_altitude_angle: float = None,
+        plot=False,
     ):
         """Determines whether a position is in line of sight of this actor
 
         Args:
             other_actor (BaseActor): The actor to check line of sight with
             epoch (pk,.epoch): Epoch at which to check the line of sight
+            minimum_altitude_angle(float): The altitude angle (in degree) at which the actor
+            has to be in relation to the surface, has to be between 0 and 90.
```

Comment:
https://rhodesmill.org/skyfield/positions.html#azimuth-and-altitude-from-a-geographic-position

horizon angle , I don't know exactly how skyfield implements this internally

Reply:
I think it is the ground station position though?

Reply:
Yes. Indeed,  I just wanted to emphasize the dependency on the position because they use a geodetic model of the Earth. So, I would assume that you don't have any spherical symmetry.

Reply:
Can you suggest a concrete change? I don't know what you want me to add

Reply:
The altitude angle (in degree) at which the actor has to be in relation to the ground station position. It has to be between 0 and 90.